drafts/SC1-2-2-captions-prerecorded.md

```diff
@@ -0,0 +1,171 @@
+# SC1-2-2-captions-prerecorded
+
+## Description
+This test checks that prerecorded synchronized media (almost video) contains captions, which are a complete substitute of the audio content.
+
+## Background
+- [Captions (Prerecorded): Understanding SC 1.2.2](http://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-captions.html)
+* [G93: Providing open (always visible) captions](http://www.w3.org/TR/2015/NOTE-WCAG20-TECHS-20150226/G93)
+* [G87: Providing closed captions](http://www.w3.org/TR/2015/NOTE-WCAG20-TECHS-20150226/G87)
+* [F8: Failure of Success Criterion 1.2.2 due to captions omitting some dialogue or important sound effects](http://www.w3.org/TR/2015/NOTE-WCAG20-TECHS-20150226/F8)
+* [F75: Failure of Success Criterion 1.2.2 by providing synchronized media without captions when the synchronized media presents more information than is presented on the page](http://www.w3.org/TR/2015/NOTE-WCAG20-TECHS-20150226/F75)
+* [F74: Failure of Success Criterion 1.2.2 and 1.2.8 due to not labeling a synchronized media alternative to text as an alternative](http://www.w3.org/TR/2015/NOTE-WCAG20-TECHS-20150226/F74)
+
+## Assumptions
+- The user defines if this test case is applicable to content that is not a video but animated
+- The synchronized media is not a media alternative for another media
+- The test does not check for programmatic determinability of the media alternative or a reference such as "The description can be found in text below" from the short text alternative of the video to it.
+
+## Test properties
+| Property                  | Value
+|---------------------------|------------------------------------------|
+| Test name                 | Captions on prerecorded synchronized media
+| Test requirement          | SC1-2-2-captions-prerecorded
+| Test mode                 | manual
+| Test environment          | rendered page + server connection
+| Test subject              | single web page
+| User expertise and skills | no prior knowledge
+| User profile              | Requires vision + hearing
+## Test procedure
+
+### Selector
+Test method: [manual]
+
+All pages including elements capable of playing video.
+
+Those can be identified by including elements having an attribute value containing a media file format, such as  .mov, .rm, .ram, .asf, .swf, .dcr, .avi, .mpg, .mpeg, .mp4, .m4v, .webm, .flv, .isma
+
+    XPath: //*[contains(@*,'.mov') or contains(@*,'.rm') or ... ]
+
+### Step 1
+Test method: [manual]
+
+Check that the content is a video containing an audio-track.
+
+**User Input Question:**
+
+| Property             | Value
+|----------------------|-------
+| presented-item       | Media content
+| requires-context     | yes
+| requires-interaction | yes
+| question             | Does the media consist of video combined with audio?
```

Comment:
Alister: Maybe a good first question would be not to look at "audio" but to look specifically for dialog.